app/views/compute_resources_vms/form/_openstack.html.erb

```diff
@@ -12,7 +12,7 @@
 </div>
 <%= select_f f, :tenant_id, compute_resource.tenants, :id, :name, {}, :label => _('Tenant') %>
 
-<%= select_f f, :security_groups, compute_resource.security_groups, :name, :name, {}, :label => _("Security group")  %>
+<%= select_f f, :security_group, compute_resource.security_groups, :name, :name, {}, :label => _("Security group")  %>
```

Comment:
I don't know Fog well enough to say, but it looks this way.  I don't really have the time to invest any time soon, do you have someone who knows this better?

All I know is it's broken as is, and works with this fix.